Can't build basic main.go from upstream

This one comes from.... https://github.com/caddyserver/caddy/blob/master/cmd/caddy/main.go:
```
❯ cat main.go
package main

import (
	caddycmd "github.com/caddyserver/caddy/v2/cmd"

	// plug in Caddy modules here
	_ "github.com/caddyserver/caddy/v2/modules/standard"
)

func main() {
	caddycmd.Main()
}
❯ xcaddy build
2020/05/07 10:23:39 [INFO] Temporary folder: /tmp/buildenv_2020-05-07-1023.089339309
2020/05/07 10:23:39 [INFO] Writing main module: /tmp/buildenv_2020-05-07-1023.089339309/main.go
2020/05/07 10:23:39 [INFO] Initializing Go module
2020/05/07 10:23:39 [INFO] exec (timeout=10s): /usr/bin/go mod init caddy 
go: creating new go.mod: module caddy
2020/05/07 10:23:39 [INFO] Pinning versions
2020/05/07 10:23:39 [INFO] exec (timeout=5m0s): /usr/bin/go get -d -v github.com/caddyserver/caddy/v2 
2020/05/07 10:23:40 [INFO] Build environment ready
2020/05/07 10:23:40 [INFO] Building Caddy
2020/05/07 10:23:40 [INFO] exec (timeout=5m0s): /usr/bin/go build -o /home/fzipi/go/src/github.com/fzipi/caddy-modsec/caddy -ldflags -w -s -trimpath 
# github.com/caddyserver/certmagic
/home/fzipi/go/pkg/mod/github.com/caddyserver/certmagic@v0.10.12/handshake.go:201:18: hello.SupportsCertificate undefined (type *tls.ClientHelloInfo has no field or method SupportsCertificate)
2020/05/07 10:23:42 [INFO] Cleaning up temporary folder: /tmp/buildenv_2020-05-07-1023.089339309
2020/05/07 10:23:42 [FATAL] exit status 2
```

I reckon this is surely from upstream.

--- a/s3www/Dockerfile
+++ b/s3www/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.13-alpine3.10
+FROM golang:1.14-alpine3.11
 
 ENV ENDPOINT http://127.0.0.1:9000
 ENV ACCESSKEY xtesting

